src/utils/validation.ts

```diff
@@ -0,0 +1,30 @@
+export const EMAIL_REGREX = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/;
+export const PASSWORD_REGREX = /^(.{1,7}|[a-zA-Z]+|\d+)$/;
```

Comment:
오타가 있네요!
사소한 오타라도 추후 유지보수에 예상 외의 영향을 줄 수 있기 때문에 유의해 주세요!
```suggestion
export const EMAIL_REGEX = /^[a-zA-Z0-9._-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/;
export const PASSWORD_REGEX = /^(.{1,7}|[a-zA-Z]+|\d+)$/;
```